modules/installing-ocp-agent-disk-partition.adoc

```diff
@@ -0,0 +1,103 @@
+// Module included in the following assemblies:
+//
+// * installing-with-agent/installing-with-agent-based-installer.adoc
+
+:_content-type: PROCEDURE
+[id="installing-ocp-agent-disk-partition_{context}"]
+= Partitioning the disk
+
+In general, you should use the default disk partitioning that is created during the {op-system} installation. However, there are cases where you might want to create a separate partition for a directory that you expect to grow.
+
+{product-title} supports the addition of a single partition to attach
+storage to either the `/var` directory or a subdirectory of `/var`.
+For example:
+
+* `/var/lib/containers`: Holds container-related content that can grow
+as more images and containers are added to a system.
+* `/var/lib/etcd`: Holds data that you might want to keep separate for purposes such as performance optimization of etcd storage.
+* `/var`: Holds data that you might want to keep separate for purposes such as auditing.
++
+[IMPORTANT]
+====
+For disk sizes larger than 100GB, and especially larger than 1TB, create a separate `/var` partition.
+====
+
+Storing the contents of a `/var` directory separately makes it easier to grow storage for those areas as needed and reinstall {product-title} at a later date and keep that data intact. With this method, you will not have to pull all your containers again, nor will you have to copy massive log files when you update systems.
+
+The use of a separate partition for the `/var` directory or a subdirectory of `/var` also prevents data growth in the partitioned directory from filling up the root file system.
+
+The following procedure sets up a separate `/var` partition by adding a machine config manifest that is wrapped into the Ignition config file for a node type during the preparation phase of an installation.
+
+.Procedure
+
+. On your installation host, change to the directory that contains the {product-title} installation program and generate the Kubernetes manifests for the cluster:
++
+[source,terminal]
+----
+$ openshift-install create manifests --dir <installation_directory>
```

Comment:
This is the one part you don't want to do in the agent-based installer. All we want here is:

```
$ mkdir <installation_directory>/openshift
```